src/components/UI/FormStyled/Form.styled.js

```diff
@@ -1,57 +1,57 @@
 import styled from 'styled-components';
 
 export const Container = styled.div`
-	color: black;
 	display: flex;
 	flex-direction: column;
+	color: black;
 	justify-content: start;
+	margin-top: 7rem;
 	text-align: start;
 	width: 100vw;
-	margin: 20px;
-	padding: 100px;
-	background-color: #694370;
+	background-color: black;
 `;
 
 export const Formfield = styled.input`
 	display: flex;
 	align-items: flex-start;
 	flex-direction: column;
-	color: white;
+	color: black;
 	font-weight: bold;
 	display: flex;
 	flex-wrap: wrap;
-	height: 50px;
+	height: 3rem;
+	margin-top: 2rem;
 	width: 80vw;
-	margin: 20px;
-	background-color: #2a2a2a;
+	margin-left: 10vw;
+	background-color: whitesmoke;
+	border: solid palevioletred;
+	border-width: 0.3rem;
+	position: relative;
+	margin-bottom: 2rem;
```

Comment:
instead of using margin here, you should give your flex container the property `gap: 2rem`

Reply:
fixed